### PR TITLE
IMPROVE - Display a Launch Screen #191

### PR DESCRIPTION
**Pull request by mitaligondaliya**
# What it Does
* IMPROVE - Display a Launch Screen #191
* Added launch screen view

# How I Tested
* Run the application
* Initially launch screen will appear with selected app icon for 2.0 sec

# Notes
* Added new view
* Inside the main app view call the launch view first
* launch view has calculation for 2 sec to show launch screen after that main tab view will display

# Screenshot

https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/29226517/b76c8605-b04f-47f8-81a8-0c1346eeac9e

### CODE REVIEW

File: Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance-WidgetExtension.xcscheme [hunk 1]
```diff
@@ -90,7 +90,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES"
-      askForAppToLaunch = "Yes"
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance-WidgetExtension.xcscheme:93]: This shouldn't have been changed, Xcode could've changed it, but next time don't commit it

**Review by mikaelacaron — APPROVED** [review]
Thanks for contributing, I'll accept this for Hacktoberfest, but I'm not going to merge this because I don't want it implemented in this way as a timer. I did some research and now understand what you meant about it not working dynamically

Next time if you could link me to some docs that would've been helpful too to know about what you meant, thanks!

**Comment by mitaligondaliya** [comment]
> Thanks for contributing, I'll accept this for Hacktoberfest, but I'm not going to merge this because I don't want it implemented in this way as a timer. I did some research and now understand what you meant about it not working dynamically
> 
> Next time if you could link me to some docs that would've been helpful too to know about what you meant, thanks!

Okay 👍🏼

**Comment by mikaelacaron** [comment]
Closing now that Hacktoberfest is over, and I don't want to merge this solution right now. Thanks again for contributing! It's much appreciated! Feel free to pick up any issues even though Hacktoberfest is over

**Pull request closed without merging**